docs/pages/reference/architecture/relay.mdx

```diff
@@ -0,0 +1,110 @@
+
+This alternate connectivity is beneficial in situations where the client and the resource are known to be close together in a physical or logical sense (the same data center, office, campus, geographical region) and regular connections through the control plane would incur higher latency, lower throughput, or higher costs.
+
+Unlike the Proxy service, the Relay does not host a web UI, intercept connections, impersonate users or provide access to the Teleport control plane API. As such, it's simpler to deploy and keep secure, and it's thus suitable to be deployed even in environments where it wouldn't be possible to securely deploy a Proxy instance.
+
+<Admonition type="warning">
+
+The Teleport Relay service is intended for specific scenarios where clients and agents are in the same network segment and there is a need for connectivity that does not go through the Teleport control plane. It is not a required or recommended cluster component in most Teleport deployments.
+
+</Admonition>
+
+## How it works
+
+A Relay service deployment consists of one or more Relay instances configured with the same Relay group name, reachable by Teleport agents and clients through a L4 network load balancer. Relay instances of the same group receive connections from agents and clients through the load balancer, and can connect to each other directly. Relay instances listen on two separate ports to serve connections from agents and clients, and a third listening port is used for direct connections from other Relay instances in the same group.
+
+The typical setup uses a load balancer serving the two ports, reachable at a single hostname, with port 3042 used by agents and port 443 used by clients. However, it's possible to use different load balancers and hostnames for the client and agent ports if necessary. The listening port for connectivity between Relay instances does not make use of the load balancer, since each Relay instance might require connecting to specific other instances to serve connections; as such, the Relay service requires outbound connectivity from each Relay instance to its peers on that port.
+
+The connections from the client to the Relay, the connections between Relay instances, and the tunnel connections from the Teleport agents and the Relay instances are all encrypted and authenticated through mutual TLS, with X.509 certificates issued by the Teleport control plane. The tunnel connections from agents to the Relay service and the connections between Relay instances use a bespoke protocol, the Relay configuration discovery from agents and the SSH connections from clients use gRPC. The use of external TLS termination is not supported.
```

Comment:
> The tunnel connections from agents to the Relay service and the connections between Relay instances use a bespoke protocol, the Relay configuration discovery from agents and the SSH connections from clients use gRPC. 

Do you think we should go into this level of detail here? Is this something that customers will care about or would this be something that only needs to be known internally?